### PR TITLE
Add Sass

### PR DESCRIPTION
**Pull request by tandavas**
**https://github.com/Famolus/awesome-scss**

**SCSS and Sass are tools that have been adopted and used by companies and projects worldwide. It would be a good idea to have a list of awesome Sass and SCSS frameworks, libraries, style guides, articles, and resources for people to discover more awesome things.**
# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!
- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. _(For example: `Add Node.js`)_
- The list I added
  - [x] **has been around for at least 20 days,**
  - [x] is a non-generated Markdown file in a GitHub repo,
  - [x] is not a duplicate,
  - [x] only has awesome items with descriptions _(Awesome lists are curations of the best, not of everything)_,
  - [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
  - [x] has a Table of Contents section named `Contents` as the first section,
  - [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) _(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)_,
  - [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
  - [x] has been checked for inconsistent formatting, spelling, and grammar _(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)_,
  - [x] has been added at the bottom of the appropriate category.

### CODE REVIEW

**Comment by sindresorhus** [comment]
I think it would be better to name the list awesome-sass. SCSS is just the syntax variant. Sass is the project name.

**Comment by tandavas** [comment]
@sindresorhus sure! Changed to awesome-sass 😄

File: readme.md [hunk 1]
```diff
@@ -188,6 +188,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [choo](https://github.com/YerkoPalma/awesome-choo) - App framework.
 - [Redux](https://github.com/brillout/awesome-redux) - State container for JavaScript apps.
 - [webpack](https://github.com/d3viant0ne/awesome-webpack) - Module bundler.
+- [Sass](https://github.com/Famolus/awesome-sass)
```
Comment (sindresorhus) [inline readme.md]: Needs a succinct description of what Sass is. See above on how to format it.

**Comment by sindresorhus** [comment]
Feedback:

- I'd like to see some links to community resources: Stack Overflow tag, IRC, chat, forums, communities, etc. Example: https://github.com/sindresorhus/awesome-electron#community
- Drop all the `A` / `An` prefixes in the descriptions.
- There should be a succinct summary at the top about what Sass is, where `Sass` is linkified to its website.
- Could use a `Tools` section for things like `libsass`.
- Would also like to see some books, videos, talks, etc.
- I don't think it makes sense to combine `Articles` and `Resources`. Should be two separate sections.

**Comment by tandavas** [comment]
Thank you for the feedback. I'll work on those things!

**Comment by tandavas** [comment]
@sindresorhus ping for another review. Sorry for the messed up commits 😢 

- [x] I'd like to see some links to community resources: Stack Overflow tag, IRC, chat, forums, communities, etc. Example: https://github.com/sindresorhus/awesome-electron#community
- [x] Drop all the A / An prefixes in the descriptions.
- [x] There should be a succinct summary at the top about what Sass is, where Sass is linkified to its website.
- [x] Could use a Tools section for things like libsass.
- [x] Would also like to see some books, videos, talks, etc.
- [x] I don't think it makes sense to combine Articles and Resources. Should be two separate sections.

Also, solved merge conflict.

**Comment by sindresorhus** [comment]
Look at the diff: https://github.com/sindresorhus/awesome/pull/814/files

**Comment by tandavas** [comment]
@sindresorhus fixed!

**Pull request merged**